lagom1-scala-conductr-bundle-lib/src/test/scala/com/typesafe/conductr/bundlelib/lagom/scaladsl/ConductRServiceLocatorSpecWithEnv.scala

```diff
@@ -38,6 +39,8 @@ class ConductRServiceLocatorSpecWithEnv extends AkkaUnitTestWithFixture("Conduct
       override lazy val circuitBreakerMetricsProvider: CircuitBreakerMetricsProvider = new CircuitBreakerMetricsProviderImpl(actorSystem)
       override lazy val circuitBreakerConfig: CircuitBreakerConfig = new CircuitBreakerConfig(configuration)
       override lazy val circuitBreakers: CircuitBreakers = new CircuitBreakers(actorSystem, circuitBreakerConfig, circuitBreakerMetricsProvider)
+
+      override lazy val serviceInfo: ServiceInfo = ServiceInfo("conductr-service.-est", Map.empty[String, immutable.Seq[ServiceAcl]])
```

Comment:
Is "-est" like ["plane-arium"](http://southpark.cc.com/clips/104356/the-plane-arium)? 😄